R/generateRmdCodeDiffExpPhylo.R

```diff
@@ -296,4 +296,166 @@ lengthNorm.limma.createRmd <- function(data.path, result.path, codefile, norm.me
   writeLines("sessionInfo()", codefile)
   writeLines("```", codefile)
   close(codefile)
+}
+
+#' Generate a \code{.Rmd} file containing code to perform differential expression analysis with length normalized counts + limma
```

Comment:
Can we modify this so that it is clear how this function is different from `lengthNorm.limma.createRmd()` (also the description below is identical between the two functions).